cabinets/__init__.py

```diff
@@ -148,3 +148,16 @@ def delete(uri: Union[str, Path], **kwargs: Any):
     """
     cabinet_, path = from_uri(uri)
     return cabinet_.delete(path, **kwargs)
+
+
+def ls(directory_uri: Union[str, Path], **kwargs: Any):
```

Comment:
So I'm not sure where I stand on this, but I'm sort of thinking `ls`  is inconsistent with the other method names. I feel like `list` is maybe more appropriate, and I like the explicit terminology. That being said, `ls` is common parlance and implies a bit more of a utility command than an operation, so I could be persuaded. We could also go a different direction with something like `contents()`, which maybe meshes a bit more with the theme of Cabinets, would be curious to hear your thoughts.

Reply:
I was super back and forth on `list` as well. I tend to agree that `list` fits much better with the rest of the method names, and now that I'm thinking about it, I think `ls` implies more functionality / similarity to Unix `ls` than this method intends to be. I'll change it to `list`.